docker/verify-c-common.Dockerfile

```diff
@@ -1,6 +1,8 @@
 
+FROM leonsou/verify-c-common:klee as klee_base
```

Comment:
all of this does not belong. I saw that you removed it in the next commit. But there is no need for the change in the first place. Please clean up PR history.

In PR, please brake changes into multiple commits. There are some changes here that relate to docker and are easy to review. Others that do the same thing to many many files, they are harder to review. If things are in separate commits it is easier for everyone to manage.